### DISCUSSION TITLE
Re-Cachine Latents every launch?

### DISCUSSION THREAD

**Discussion by rafstahelin** [bmaltais/kohya_ss#2247] [Q&A]
I have `Cache latents` and `Cache latents to disk` checked in Finetuning module. Every time I launch training, looks like caching takes place again. This wasnt happening on Lora module. Is there a reason? I am running on Runpod, Kohya latest release 23.1.3

**Comment by SarahPeterson2854** [comment]
bump, possibly related to captions json
